x-pack/plugins/triggers_actions_ui/public/types.ts

```diff
@@ -95,6 +96,7 @@ export interface AlertTypeModel {
   validate: (alertParams: any) => ValidationResult;
   alertParamsExpression: React.FunctionComponent<any>;
   defaultActionMessage?: string;
+  defaultActionGroup?: string;
```

Comment:
I'm thinking this would belong nicely next to `actionGroups` on the server side? here: https://github.com/elastic/kibana/blob/master/x-pack/plugins/alerting/server/types.ts#L66. Then we return the defaultActionGroup with the `/types` API. Thoughts?

Reply:
Good point. I will update the PR the way you proposed.

Reply:
Make it optional?  And then the same logic to return the first one, if not specified?  Probably makes sense to do that, as long as we can do it in one place; we shouldn't repeat that default calculation on the client, if we can.